### PR TITLE
User- and file-related endpoints

### PR DESCRIPTION
**Pull request by VinyarionHyarmendacil**
`GET /auth/user/favoritelimits`
`GET /analysis/{fileId}/{versionId}`
`GET /analysis/{fileId}/{versionId}/security`
`GET /analysis/{fileId}/{versionId}/standard`
`GET /users/{userId}/feedback`
`GET /userNotes`
`POST /userNotes`
`GET /userNotes/{userNoteId}`
+ fixed incorrect example request method
+ fixed typo in path variable

### CODE REVIEW

**Comment by jellejurre** [comment]
Looks good

Do you think you could add relevant tests to the spec tests for this one as well?

File: openapi/components/paths/users.yaml [hunk 1]
```diff
@@ -194,6 +194,85 @@ paths:
           $ref: ../responses/MissingCredentialsError.yaml
       security:
         - authCookie: []
+  '/users/{userId}/feedback':
```
Comment (Hackebein) [inline openapi/components/paths/users.yaml:197]:
```suggestion
  '/users/{userId}/feedback':
    depricated: true
```

Comment (VinyarionHyarmendacil) [reply openapi/components/paths/users.yaml:197]: spec liter complains

**Comment by VinyarionHyarmendacil** [comment]
@jellejurre None of the responses I get contain the `targetUser` property; I don't think it should be required by default.

File: openapi/components/paths/users.yaml [hunk 2]
```diff
@@ -194,6 +194,85 @@ paths:
...
+    parameters:
+      - $ref: ../parameters.yaml#/userId
+      - $ref: ../parameters.yaml#/contentId
+      - $ref: ../parameters.yaml#/number
+      - $ref: ../parameters.yaml#/offset
+    get:
```
Comment (Hackebein) [inline openapi/components/paths/users.yaml:204]:
```suggestion
    get:
      depricated: true
```

**Comment by jellejurre** [comment]
yeah I noticed that as well, but it definitely got added once. Don't know what causes it to get added, but making it optional for now

File: openapi/components/paths/users.yaml (hunk 1 above)
Comment (Hackebein) [reply openapi/components/paths/users.yaml:197]: you are right. was on the wrong path. Added another comment

**Comment by jellejurre** [comment]
LGTM. You okay with my changes here and in the spec tests? @VinyarionHyarmendacil

**Comment by VinyarionHyarmendacil** [comment]
ye👍

**Review by jellejurre — APPROVED** [review]

**Pull request merged**